docs/api/text/freqdist.rst

```diff
@@ -120,11 +127,41 @@ And for the gaming corpus (again oriented horizontally):
     visualizer.fit(docs)
     visualizer.show()
 
+Quick Method
+------------
+
+Similar functionality as above can be achieved in one line using the associated quick method, ``freqdist``. This method will instantiate with features(words) and fit a ``FreqDistVisualizer`` visualizer on the documents).
+
+.. plot::
+    :context: close-figs
+    :alt: Frequency Distribution quick method
+
+    from collections import defaultdict
+
+    from sklearn.feature_extraction.text import CountVectorizer
+
+    from yellowbrick.text import freqdist
+    from yellowbrick.datasets import load_hobbies
+
+    # Load the text data
+    corpus = load_hobbies()
+
+    # Create a dict to map target labels to documents of that category
+    hobbies = defaultdict(list)
+    for text, label in zip(corpus.data, corpus.target):
+        hobbies[label].append(text)
+
+    vectorizer = CountVectorizer(stop_words='english')
+    docs       = vectorizer.fit_transform(text for text in hobbies['cinema'])
+    features   = vectorizer.get_feature_names()
+
+    freqdist(features, docs, orient='v')
```

Comment:
Great illustration of what the quick method can do! I only tweaked the example slightly to add the `orient` parameter in the interest of showing users that all the original `FreqDistVisualizer` params still work!